coalib/parsing/DefaultArgParser.py

```diff
@@ -170,6 +170,10 @@ def default_arg_parser(formatter_class=None):
             help="filters `--show-bears` by the given languages")
 
         outputs_group.add_argument(
+           '-x', '--show-capabilites', nargs='+', metavar='LANG',
```

Comment:
why x? Nothing else left? Any upper case letter that relates to it somehow more than x?